app/packs/src/apps/mydb/elements/details/researchPlans/ResearchPlanDetails.js

```diff
@@ -481,7 +483,16 @@ export default class ResearchPlanDetails extends Component {
   } /* eslint-enable */
 
   renderPanelHeading(researchPlan) {
+    const { currentCollection } = UIStore.getState();
+    const defCol = currentCollection && currentCollection.is_shared === false &&
```

Comment:
maybe extract as method with a nice name or rename variable to make more clear that this is the collection we want to copy the RP into.